dredd/api-description.yml

```diff
@@ -2342,6 +2370,13 @@ parameters:
     description: The episode id to retrieve. E.g. s02e03, e34 or 2016-12-31
     x-example: s01e01
     type: string
+  episode-id-delete:
+    name: episode-id
```

Comment:
Does this also need `-delete`?

Reply:
What I mean is the ones above all have it matching with their name where as this one is missing the `-delete`. Might be related.

Reply:
Oh like that, i dont think the name makes any difference.